---
author-meta:
- "Kim-Anh L\xEA Cao"
- Aedin C Culhane
- Elana Fertig
- Jane Roe
bibliography:
- content/manual-references.json
date-meta: '2020-07-01'
header-includes: "<!--\nManubot generated metadata rendered from header-includes-template.html.\nSuggest improvements at https://github.com/manubot/manubot/blob/master/manubot/process/header-includes-template.html\n-->\n<meta name=\"dc.format\" content=\"text/html\" />\n<meta name=\"dc.title\" content=\"Mathematical Frameworks for Integrative Analysis of Multi-omics Biological Data\" />\n<meta name=\"citation_title\" content=\"Mathematical Frameworks for Integrative Analysis of Multi-omics Biological Data\" />\n<meta property=\"og:title\" content=\"Mathematical Frameworks for Integrative Analysis of Multi-omics Biological Data\" />\n<meta property=\"twitter:title\" content=\"Mathematical Frameworks for Integrative Analysis of Multi-omics Biological Data\" />\n<meta name=\"dc.date\" content=\"2020-07-01\" />\n<meta name=\"citation_publication_date\" content=\"2020-07-01\" />\n<meta name=\"dc.language\" content=\"en-US\" />\n<meta name=\"citation_language\" content=\"en-US\" />\n<meta name=\"dc.relation.ispartof\" content=\"Manubot\" />\n<meta name=\"dc.publisher\" content=\"Manubot\" />\n<meta name=\"citation_journal_title\" content=\"Manubot\" />\n<meta name=\"citation_technical_report_institution\" content=\"Manubot\" />\n<meta name=\"citation_author\" content=\"Kim-Anh L\xEA Cao\" />\n<meta name=\"citation_author_institution\" content=\"Melbourne Integrative Genomics, School of Mathematics and Statistics, University of Melbourne, Australia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0003-3923-1116\" />\n<meta name=\"twitter:creator\" content=\"@mixOmicsTeam\" />\n<meta name=\"citation_author\" content=\"Aedin C Culhane\" />\n<meta name=\"citation_author_institution\" content=\"Data Sciences, Dana-Farber Cancer Institute, Boston, MA, USA\" />\n<meta name=\"citation_author_institution\" content=\"Biostatsitics, Harvard TH Chan School of Public Health, Boston, MA, USA\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-1395-9734\" />\n<meta name=\"twitter:creator\" content=\"@AedinCulhane\" />\n<meta name=\"citation_author\" content=\"Elana Fertig\" />\n<meta name=\"citation_author_institution\" content=\"Department of Oncology, Sidney Kimmel Comprehensive Cancer Center, Johns Hopkins University School of Medicine, Baltimore, MD, USA\" />\n<meta name=\"citation_author_institution\" content=\"Department of Biomedical Engineering, Johns Hopkins University School of Medicine, Baltimore, MD, USA\" />\n<meta name=\"citation_author_institution\" content=\"Department of Applied Mathematics and Statistics, Johns Hopkins University Whiting School of Engineering, Baltimore, MD, USA\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0003-3204-342X\" />\n<meta name=\"twitter:creator\" content=\"@FertigLab\" />\n<meta name=\"citation_author\" content=\"Jane Roe\" />\n<meta name=\"citation_author_institution\" content=\"Department of Something, University of Whatever\" />\n<meta name=\"citation_author_institution\" content=\"Department of Whatever, University of Something\" />\n<meta name=\"citation_author_orcid\" content=\"XXXX-XXXX-XXXX-XXXX\" />\n<meta name=\"twitter:creator\" content=\"@XXX\" />\n<link rel=\"canonical\" href=\"https://BIRSBiointegration.github.io/whitePaper/\" />\n<meta property=\"og:url\" content=\"https://BIRSBiointegration.github.io/whitePaper/\" />\n<meta property=\"twitter:url\" content=\"https://BIRSBiointegration.github.io/whitePaper/\" />\n<meta name=\"citation_fulltext_html_url\" content=\"https://BIRSBiointegration.github.io/whitePaper/\" />\n<meta name=\"citation_pdf_url\" content=\"https://BIRSBiointegration.github.io/whitePaper/manuscript.pdf\" />\n<link rel=\"alternate\" type=\"application/pdf\" href=\"https://BIRSBiointegration.github.io/whitePaper/manuscript.pdf\" />\n<link rel=\"alternate\" type=\"text/html\" href=\"https://BIRSBiointegration.github.io/whitePaper/v/8b5dc2399e2a88c12b6021883cc5f194bf84fd2a/\" />\n<meta name=\"manubot_html_url_versioned\" content=\"https://BIRSBiointegration.github.io/whitePaper/v/8b5dc2399e2a88c12b6021883cc5f194bf84fd2a/\" />\n<meta name=\"manubot_pdf_url_versioned\" content=\"https://BIRSBiointegration.github.io/whitePaper/v/8b5dc2399e2a88c12b6021883cc5f194bf84fd2a/manuscript.pdf\" />\n<meta property=\"og:type\" content=\"article\" />\n<meta property=\"twitter:card\" content=\"summary_large_image\" />\n<link rel=\"icon\" type=\"image/png\" sizes=\"192x192\" href=\"https://manubot.org/favicon-192x192.png\" />\n<link rel=\"mask-icon\" href=\"https://manubot.org/safari-pinned-tab.svg\" color=\"#ad1457\" />\n<meta name=\"theme-color\" content=\"#ad1457\" />\n<!-- end Manubot generated metadata -->"
keywords:
- markdown
- publishing
- manubot
lang: en-US
manubot-clear-requests-cache: false
manubot-output-bibliography: output/references.json
manubot-output-citekeys: output/citations.tsv
manubot-requests-cache-path: ci/cache/requests-cache
title: Mathematical Frameworks for Integrative Analysis of Multi-omics Biological Data
...






<small><em>
This manuscript
([permalink](https://BIRSBiointegration.github.io/whitePaper/v/8b5dc2399e2a88c12b6021883cc5f194bf84fd2a/))
was automatically generated
from [BIRSBiointegration/whitePaper@8b5dc23](https://github.com/BIRSBiointegration/whitePaper/tree/8b5dc2399e2a88c12b6021883cc5f194bf84fd2a)
on July 1, 2020.
</em></small>

## Authors



+ **Kim-Anh Lê Cao**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-3923-1116](https://orcid.org/0000-0003-3923-1116)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [mixOmicsTeam](https://github.com/mixOmicsTeam)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [mixOmicsTeam](https://twitter.com/mixOmicsTeam)<br>
  <small>
     Melbourne Integrative Genomics, School of Mathematics and Statistics, University of Melbourne, Australia
     · Funded by Grant National Health and Medical Research Council  Career Development fellowship (GNT1159458)
  </small>

+ **Aedin C Culhane**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-1395-9734](https://orcid.org/0000-0002-1395-9734)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [aedin](https://github.com/aedin)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [AedinCulhane](https://twitter.com/AedinCulhane)<br>
  <small>
     Data Sciences, Dana-Farber Cancer Institute, Boston, MA, USA; Biostatsitics, Harvard TH Chan School of Public Health, Boston, MA, USA
     · Funded by Chan Zuckerberg Initative, NIH, DoD (need to get grant IDs)
  </small>

+ **Elana Fertig**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-3204-342X](https://orcid.org/0000-0003-3204-342X)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [ejfertig](https://github.com/ejfertig)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [FertigLab](https://twitter.com/FertigLab)<br>
  <small>
     Department of Oncology, Sidney Kimmel Comprehensive Cancer Center, Johns Hopkins University School of Medicine, Baltimore, MD, USA; Department of Biomedical Engineering, Johns Hopkins University School of Medicine, Baltimore, MD, USA; Department of Applied Mathematics and Statistics, Johns Hopkins University Whiting School of Engineering, Baltimore, MD, USA
     · Funded by National Institute of Health, National Cancer Institute; National Institute of Health, National Institute of Dental and Craniofacial Research; Lustgarten Foundation; Emerson Foundation; Allegheny Health Network
  </small>

+ **Jane Roe**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [XXXX-XXXX-XXXX-XXXX](https://orcid.org/XXXX-XXXX-XXXX-XXXX)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [janeroe (PLEASE COPY/PASTE DO NOT EDIT THIS ONE)](https://github.com/janeroe (PLEASE COPY/PASTE DO NOT EDIT THIS ONE))
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [XXX](https://twitter.com/XXX)<br>
  <small>
     Department of Something, University of Whatever; Department of Whatever, University of Something
     · Funded by XX
  </small>



## Abstract {.page_break_before}




## Introduction 

### Multi-omics and multi-view single cell technologies

- Do multi-omics answer our biological questions?
    Single cell community has naturally adopted a data-driven approach
    Helpful to complement with hypothesis-driven and mechanistic-driven
    approaches
    Technological advances will help refine our hypothesis (e.g. multi-modal
    studies focusing on a set of specific genes)
    Can tell us how different levels of regulation are influencing each other


- State-of-the-art

Brief overview of our three hackathon studies


- What do we hope to learn from multi-omics / multi view single cell studies

What bulk TCGA has taught us:
    - Type of omics that can answer a specific biological question
    - The value of open resources for methodological developments
    - New hypotheses

New single cell multi omics initiatives:
    - Human Cell Atlas (HCA): assess variation in normal tissues
    - Human Tumor Atlas Network (HTAN):
      Clinical, experimental, computational framework to generate three- dimensional atlases of cancer transitions for diverse tumor types. Single-cell, longitudinal, and clinical outcomes

-  Are our technologies ready? 

  Do technologies capture the information we want?
  Our hackathons featuing cutting edge technologies have showed that:
    - Issues with noise and experimental design
    - Time lag between regulatory levels not addressed and many open questions
remain (e.g methylation / gene expression)
    - Direction of regulation not captured
    
    
- **Objectives of this paper**
    - Provide guidelines on tools / data / technologies / methods and needs to model the multi-scale regulatory processes in biological systems for a computational biologist audience
    
    
- **Outline and messages**
    - Cellular and molecular regulation is fundamentally multi-scale and captured by distinct data modalities
    - Traditional hypothesis-driven multi-omics/view studies only consider one facet of these technologies, but more can be learned through a holistic approach extending into atlases
    - We present our learnings from some hackathons case studies that helped us obtain a broader picture and language
    
    INCLUDE FIGURE 1 OUTLINE





### scRNA-seq + FISH as a case study for spatial transcriptomics

#### Overview and biological question

#### Computational challenges

* Can scRNA-seq data be overlaid onto seqFISH for resolution enhancement
* What is the minimal number of genes needed for data integration?
* Are there signatures of cellular co-localization or spatial coordinates in the non-spatial scRNA-seq data?

#### Methods for stats/math analyses and results summary


### Spatial proteomics and cross-study analysis

#### Overview and biological question

#### Computational challenges

* Integrating partially-overlapping proteomic data collected on different patients with similar phenotypes
* Integration of spatial x-y coordinate co-location and co-expression
* Integration with other 'omics datasets (e.g., scRNA-seq) to support the results of these proteomic analyses
* Can we predict the spatial expression patterns of proteins measured on mass-tag but not measured in the MIBI-TOF data?
* What additional information can we learn about the different macrophage and immune populations in breast cancer by conducting integrated analyses of these datasets?

#### Methods for stats/math analyses and results summary


### scNMT-seq as a case-study for epigenetic regulation

#### Overview and biological question

#### Computational challenges

* Identification of multi-omics signatures that characterize lineage, stage or both.
* Handling missing values
* Do epigenetic changes in some genomic contexts affect cell fate decision more than others? If so, how?

#### Methods for stats/math analyses and results summary


## Overview of common analytical methods spanning technologies / case studies

* matrix factorization
* neural network / autoencoders


## Challenges for interpretation


### Interpretation requires a good understanding of the methods (no black boxes, a common language)

Communicating within the field: what approaches are we talking about?

Ref to Glossary of terms


### Interpretation of the output from the analyses of the data is facilitated by the incorporation of contiguous information.

Redundant biological knowledge and incorporation of information from databases are important in the process.
Biological Interpretations: bridges to data bases such as KEGG, Gene Ontology, HCA.
Validation  through complementary data.

### Visualization tools for interpretation and communication

Examples: Brushing UMAP.

### Explaining results to biologists through generative models and simulations (ex: Factor Analysis).

### Issues of over-discretization, over-simplification

Example 1: The notion of cell-type is insufficient (Communication challenge with biologists about tradeoffs between focusing on rare cell types vs. more “continuous” view on cell types).

Problem with loss of information in the desire to simplify.

Over interpretation of graphical outputs

### Counterexamples


## Techniques and challenges for benchmarking methods

### We must first define what we are benchmarking

 * Often the goal in benchmarking is recovery of known cell types with
   processing of raw data, quantification, and clustering. The
   Adjusted Rand Index (ARI) or other metrics for partitions are
   used. 
 * One may also attempt to benchmark methods for their ability to
   discover known relationships between data modalities, e.g. gene
   regulatory relationships observed between chromatin accessibility
   and gene expression. However, this is made difficult by the fact
   that these relationships are not fully known at the single cell
   level.
	
### Strategies for benchmarking

* Simulation is useful for having known truth, but it is difficult to
  simulate realistic covariance structure across features and across
  data modalities.
* Benchmarking datasets (add examples from Google Doc). Benchmark
  datasets for single cell studies have largely centered around
  measuring sequencing depth and diversity of cell types derived from
  a single assay of interest (e.g. scRNAseq). A benchmark dataset
  serves a few purposes:
    - Provides ground truth for the intended effect of exposure in a
      proposed study design.
    - Provides validation for a data integration task for which a new
      computational method may be proposed.
	
  For multi-modal assays, while the intended effects can vary based on
  the leading biological questions, one may abstract out common data
  integration tasks such as co-embedding, mapping or correlation, and
  inferring causal relationships. We distinguish data integration from
  further downstream analyses that may occur on integrated samples
  such as differential analysis of both assays wrt to a certain
  exposure.

  Both the intended effects and data integration task rely on study
  design that takes into account: 
    - Biological and technical variability via replicates, block
      design, and randomization. 
	- Power analysis for the intended effect or data integration
      task. 
	- Dependencies between modalities, for e.g. gene expression
      depending on gene regulatory element activity, requires that
      experiment design must also account for spatial and temporal
      elements in sampling for a given observation.
	
  As such, no universal benchmark data scheme may suit every
  combination of modality, and benchmark datasets may be established
  for commonly used combinations of modalities or technologies,
  towards specific data integration tasks.

* Cross-validation within study can be performed. For example the
  following cross-validation analysis of the scNMT-seq dataset was
  performed using MOFA+
  
  <img src="images/scNMT_MOFA_CV.png" alt="Cross validation of MOFA+ on scNMT-seq" height="300"/>

  A challenge with within study cross-validation is how to 
  match dimensions of latent space across folds. (add examples from
  Google Doc of papers that have performed either permutation or
  cross-validation to assess model performance)
* Cross-study validation would assess if relationships discovered in
  one dataset present in other datasets, potentially looking across
  single cell and bulk.


## Software strategies to enable analyses of multimodal single cell experiments

### Key questions 

* How should multimodal single cell data be managed for interactive and batch analyses?
* What methods will help software developers create scalable solutions for multimodal single cell analysis?
* How can we ensure that visualization methods that are central to multimodal single cell analysis
are usable by researchers with visual impairments?

### Data management strategies

* Abstract data type: "multiassay experiment".  This reflects the idea that each mode will
be characterized by a different collection of features on possibly non-overlapping collections
of samples.  The metadata on features should be clearly and conventionally defined.  For example,
genes and transcripts are enumerated using Ensembl catalog identifiers; regions of accessibility
are defined using genomic coordinates in a clearly specified reference build.  Metadata on
samples must include all relevant information on experimental conditions such as treatment,
protocol, and date of technical processing.

(More fodder-AS)

**Key points:**
1) What do we want to store and share? Data object vs analysis object. How would the design change based on what is stored?
2) Do we need a flexible, universal framework (e.g. MAE) or an experiment class for every possible combination of modalities or technologies?
3) Do we have adequate data representation for all “assays”?

   - Multi-modal single cell data may consist of multi-assay measurements from the same cell (e.g. citeSeq, sci-CAR) or integration of multi-assay measurements from distinct cells from the same or distinct starting samples. A sample here refers to the biological specimen of origin (tissue A from individual X).
A data container for a multi assay analysis must hold 
   1) Assay slots containing variables or features from multiple modalities (e.g. gene expression units from sc-RNAseq and protein units from sc-proteomics). In some cases, the feature may be multidimensional (e.g. spatial coordinates, locations of eQTLs).
   2) Observations or cell identities
   3) Metadata for sample of origin for the individual cells, e.g. study, center, phenotype, perturbation. 
   4) A map between the different assays to enable analysis

   - The MAE is such a Bioconductor container for overlapping observations, and may serve as a starting point for further expansion. Besides the primary data elements for storing “data objects”, the summarizedExperiment class offers attributes and Methods for storing results of analysis, as an “analysis object”. 
      - While common assays such as RNAseq and ATACseq have well-defined data representations (e.g. transcript names), data representation need to be defined newer assays, which may need multiple dimensions for adequate definition (e.g. x, y, z coordinates for images).
      - The observations of different modalities may not be directly comparable (e.g. RNA may be measured from individual cells but spatial transcriptomics may cover a few cells in the matched area).  
      - In the absence of universal standards, the metadata may vary from analysis to analysis.

(Note, standard BioC container/class terms may not be correctly used. End of fodder - AS)

* Serializations and data access methods for
    * spatial transcriptomics
    * scNMT-seq ...


### Scalability strategies

### Reducing barriers to interpretable visualizations
Color is a powerful data visualization tool that helps representing the different dimensions of our increasingly complex and rich scientific data. 
Color vision deficiencies affect a substantial portion of the population. 
Therefore,  it is desirable to aim towards presenting scientific information in a manner that is as accessible as possible for all readers.  
Color vision deficiency leads to difficulties in perceiving patterns (the basis for the Ishihara's color vision tests) in multi-colored figures. 
In rare cases, the perceived patterns; e.g. in heatmaps and reduced dimension plots, can differ between individuals with normal and color deficient vision. 

One strategy to address these issues is to include colorblind friendly visualizations [@https://doi.org/10.1038/nmeth.1618] as a default setting in our visualizations. 
Several colorblind-friendly palettes exist (e.g., see R packages [viridis](https://cran.r-project.org/web/packages/viridis/vignettes/intro-to-viridis.html) and [dittoSeq](https://github.com/dtm2451/dittoSeq)) and can be integrated into data presentation as the default option. 
Even with these palettes in place, it is desirable to limit the number (about 8-10 at a maximum) of colors in visualizations. 
To reduce the dependence on colors, one solution would be to include additional visual cues to differentiate regions (hatched areas) or cells (point shapes). 
Overall, a broader discussion regarding the accessibility of our figures that is not just limited to color vision deficiencies would be greatly beneficial towards improving data accessibility. 
Perhaps one tool to address broader accessibility could be the inclusion an "accessibility caption" accompanying figures which "guide" the reader's perception of the images.


[Reference 1: Color Coding](https://www.nature.com/articles/nmeth0810-573)

[Reference 2: Points of View: Color Blindness](https://www.nature.com/articles/nmeth.1618)

[Viridis Color Palettes](https://cran.r-project.org/web/packages/viridis/vignettes/intro-to-viridis.html)

[An overview of the issues with impaired color perception](https://www.vwvj.be/sites/default/files/zien/zien_-_uit_voorbije_vorming/kleurzinonderzoekextratips.pdf)

[US Government tools for accessibility](https://accessibility.18f.gov/tools/)

### Details of working components -- trimmed 

you can interact with underlying data at [google sheet](https://docs.google.com/spreadsheets/d/1tSUQ9iDKqq72TB9G3Cx1evg2sekS-ytx5wRXDv6vxfg/edit?usp=sharing)

|Type|Brief name (link)|Description|
|----|-----------------|-----------|
|Matlab package|[CytoMAP](https://gitlab.com/gernerlab/cytomap)|CytoMAP: A Spatial Analysis Toolbox Reveals Features of Myeloid Cell Organization in Lymphoid Tissues|
|Matlab package|[histoCAT](https://github.com/BodenmillerGroup/histoCAT)|histoCAT: analysis of cell phenotypes and interactions in multiplex image cytometry data|
|Python library|[PyTorch](https://pytorch.org)|General framework for deep learning|
|Python package|[SpaCell](https://github.com/BiomedicalMachineLearning/SpaCell)|SpaCell: integrating tissue morphology and spatial gene expression to predict disease cells|
|Python package|[Scanpy](https://github.com/theislab/scanpy)|Python package for single cell analysis|
|R data class|[MultiAssayExperiment](https://bioconductor.org/packages/MultiAssayExperiment)|unify multiple experiments|
|R data class|[SpatialExperiment](https://github.com/drighelli/SpatialExperiment)|SpatialExperiment: a collection of S4 classes for Spatial Data|
|R package|[Giotto](https://github.com/RubD/Giotto)|Spatial transcriptomics|
|R package|[cytomapper](https://github.com/BodenmillerGroup/cytomapper)|cytomapper: Visualization of highly multiplexed imaging cytometry data in R|
|R package|[Spaniel](https://github.com/RachelQueen1/Spaniel/)|Spaniel: analysis and interactive sharing of Spatial Transcriptomics data|
|R package|[Seurat](https://github.com/satijalab/seurat)|R toolkit for single cell genomics|
|R package|[SpatialLIBD](https://github.com/LieberInstitute/spatialLIBD)|Transcriptome-scale spatial gene expression in the human dorsolateral prefrontal cortex|
|R package|[Cardinal](https://cardinalmsi.org/)|Cardinal: an R package for statistical analysis of mass spectrometry-based imaging experiments|
|R package|[CoGAPS](https://github.com/FertigLab/CoGAPS)|scCoGAPS learns biologically meaningful latent spaces from sparse scRNA-Seq data|
|R package|[projectR](https://github.com/genesofeve/projectR)|ProjectR is a transfer learning framework to rapidly explore latent spaces across independent datasets|
|R package|[SingleCellMultiModal](https://github.com/waldronlab/SingleCellMultiModal)|Serves multiple datasets obtained from GEO and other sources and represents them as MultiAssayExperiment objects|
|R scripts|[SpatialAnalysis](https://github.com/drighelli/SpatialAnalysis)|Scripts for SpatialExperiment usage|
|Self-contained GUI|[ST viewer](https://github.com/jfnavarro/st_viewer)|ST viewer: a tool for analysis and visualization of spatial transcriptomics datasets|
|Shiny app|[Dynverse](https://zouter.shinyapps.io/server/)|A comparison of single-cell trajectory inference methods: towards more accurate and robust tools|


Here is the schematic of SpatialExperiment class from Dario Righelli.

<img src="images/SpatialExperimentScheme1.1.png" alt="SpatialExperiment (D. Righelli)" height="400"/>


Here is the schematic of how seqFISH data are stored inot the SingleCellMultiModal package from Dario Righelli.

<img src="images/scMultiModal_seqFISH.png" alt="seqFISH (D. Righelli)" height="400"/>




## Discussion

### Emerging analytical methods and technologies

### Community needs for data structures, analysis methods, etc 


## Glossary

| Consensus term | Synonyms | Description |
| -------------- | -------- | ----------- |
| Network | Graph | A set of *nodes*, representing objects of interest, linked by *edges*, representing specific relationships between nodes. |
| Node | Vertex | Element of interest in a network and linked to other nodes. For example: people, cells, proteins or genes. Nodes can have several properties called *attributes* like cell type or position. |
| Edge | Link | The relationship between 2 nodes in a network. For example: friendship in social networks, cells in contact in a spatial network, or gene-gene interactions in a gene regulatory network. |


## References {.page_break_before}

<!-- Explicitly insert bibliography here -->
<div id="refs"></div>
