---
author-meta:
- "Kim-Anh L\xEA Cao"
- Jane Roe
bibliography:
- content/manual-references.json
date-meta: '2020-04-22'
header-includes: "<!--\nManubot generated metadata rendered from header-includes-template.html.\nSuggest improvements at https://github.com/manubot/manubot/blob/master/manubot/process/header-includes-template.html\n-->\n<meta name=\"dc.format\" content=\"text/html\" />\n<meta name=\"dc.title\" content=\"Mathematical Frameworks for Integrative Analysis of Multi-omics Biological Data\" />\n<meta name=\"citation_title\" content=\"Mathematical Frameworks for Integrative Analysis of Multi-omics Biological Data\" />\n<meta property=\"og:title\" content=\"Mathematical Frameworks for Integrative Analysis of Multi-omics Biological Data\" />\n<meta property=\"twitter:title\" content=\"Mathematical Frameworks for Integrative Analysis of Multi-omics Biological Data\" />\n<meta name=\"dc.date\" content=\"2020-04-22\" />\n<meta name=\"citation_publication_date\" content=\"2020-04-22\" />\n<meta name=\"dc.language\" content=\"en-US\" />\n<meta name=\"citation_language\" content=\"en-US\" />\n<meta name=\"dc.relation.ispartof\" content=\"Manubot\" />\n<meta name=\"dc.publisher\" content=\"Manubot\" />\n<meta name=\"citation_journal_title\" content=\"Manubot\" />\n<meta name=\"citation_technical_report_institution\" content=\"Manubot\" />\n<meta name=\"citation_author\" content=\"Kim-Anh L\xEA Cao\" />\n<meta name=\"citation_author_institution\" content=\"Melbourne Integrative Genomics, School of Mathematics and Statistics, University of Melbourne, Australia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0003-3923-1116\" />\n<meta name=\"twitter:creator\" content=\"@mixOmicsTeam\" />\n<meta name=\"citation_author\" content=\"Jane Roe\" />\n<meta name=\"citation_author_institution\" content=\"Department of Something, University of Whatever\" />\n<meta name=\"citation_author_institution\" content=\"Department of Whatever, University of Something\" />\n<meta name=\"citation_author_orcid\" content=\"XXXX-XXXX-XXXX-XXXX\" />\n<link rel=\"canonical\" href=\"https://BIRSBiointegration.github.io/whitePaper/\" />\n<meta property=\"og:url\" content=\"https://BIRSBiointegration.github.io/whitePaper/\" />\n<meta property=\"twitter:url\" content=\"https://BIRSBiointegration.github.io/whitePaper/\" />\n<meta name=\"citation_fulltext_html_url\" content=\"https://BIRSBiointegration.github.io/whitePaper/\" />\n<meta name=\"citation_pdf_url\" content=\"https://BIRSBiointegration.github.io/whitePaper/manuscript.pdf\" />\n<link rel=\"alternate\" type=\"application/pdf\" href=\"https://BIRSBiointegration.github.io/whitePaper/manuscript.pdf\" />\n<link rel=\"alternate\" type=\"text/html\" href=\"https://BIRSBiointegration.github.io/whitePaper/v/6b895b4054b2066479b4269de76c1c2283cc3387/\" />\n<meta name=\"manubot_html_url_versioned\" content=\"https://BIRSBiointegration.github.io/whitePaper/v/6b895b4054b2066479b4269de76c1c2283cc3387/\" />\n<meta name=\"manubot_pdf_url_versioned\" content=\"https://BIRSBiointegration.github.io/whitePaper/v/6b895b4054b2066479b4269de76c1c2283cc3387/manuscript.pdf\" />\n<meta property=\"og:type\" content=\"article\" />\n<meta property=\"twitter:card\" content=\"summary_large_image\" />\n<link rel=\"icon\" type=\"image/png\" sizes=\"192x192\" href=\"https://manubot.org/favicon-192x192.png\" />\n<link rel=\"mask-icon\" href=\"https://manubot.org/safari-pinned-tab.svg\" color=\"#ad1457\" />\n<meta name=\"theme-color\" content=\"#ad1457\" />\n<!-- end Manubot generated metadata -->"
keywords:
- markdown
- publishing
- manubot
lang: en-US
manubot-clear-requests-cache: false
manubot-output-bibliography: output/references.json
manubot-output-citekeys: output/citations.tsv
manubot-requests-cache-path: ci/cache/requests-cache
title: Mathematical Frameworks for Integrative Analysis of Multi-omics Biological Data
...






<small><em>
This manuscript
([permalink](https://BIRSBiointegration.github.io/whitePaper/v/6b895b4054b2066479b4269de76c1c2283cc3387/))
was automatically generated
from [BIRSBiointegration/whitePaper@6b895b4](https://github.com/BIRSBiointegration/whitePaper/tree/6b895b4054b2066479b4269de76c1c2283cc3387)
on April 22, 2020.
</em></small>

## Authors



+ **Kim-Anh Lê Cao**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-3923-1116](https://orcid.org/0000-0003-3923-1116)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [mixOmicsTeam](https://github.com/mixOmicsTeam)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [mixOmicsTeam](https://twitter.com/mixOmicsTeam)<br>
  <small>
     Melbourne Integrative Genomics, School of Mathematics and Statistics, University of Melbourne, Australia
     · Funded by Grant National Health and Medical Research Council  Career Development fellowship (GNT1159458).
  </small>

+ **Jane Roe**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [XXXX-XXXX-XXXX-XXXX](https://orcid.org/XXXX-XXXX-XXXX-XXXX)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [janeroe](https://github.com/janeroe)<br>
  <small>
     Department of Something, University of Whatever; Department of Whatever, University of Something
  </small>



## Abstract {.page_break_before}




## Introduction to single cell and imaging multi-omics

Testing github pages.


## Current multi-omic technologies


## Challenges for interpretation

Need for technology-specific questions and analysis methods vs one size fits all data blender


## Case studies


### scNMT-seq as a case-study for epigenetic regulation

#### Overview and biological question

#### Computational challenges

* Identification of multi-omics signatures that characterise lineage, stage or both.
* Handling missing values
* Do epigenetic changes in some genomic contexts affect cell fate decision more than others? If so, how?

#### Methods for stats/maths analyses and results summary


### scRNA-seq + FISH as a case study for spatial transcriptomics

#### Overview and biological question

#### Computational challenges

* Can scRNA-seq data be overlaid onto seqFISH for resolution enhancement
* What is the minimal number of genes needed for data integration?
* Are there signatures of cellular co-localization or spatial coordinates in the non-spatial scRNA-seq data?

#### Methods for stats/maths analyses and results summary


### Spatial proteomics and cross-study analysis

#### Overview and biological question

#### Computational challenges

* Integrating partially-overlapping proteomic data collected on different patients with similar phenotypes
* Integration of spatial x-y coordinate co-location and co-expression
* Integration with other 'omics datasets (e.g., scRNA-seq) to support the results of these proteomic analyses
* Can we predict the spatial expression patterns of proteins measured on mass-tag but not measured in the MIBI-TOF data?
* What additional information can we learn about the different macrophage and immune populations in breast cancer by conducting integrated analyses of these datasets?

#### Methods for stats/maths analyses and results summary


## Overview of common analytical methods spanning technologies / case studies

* matrix factorization
* neural network / autoencoders


## Data structures and software packages to enable analyses

* Bioc/multiAssayExperiment for single cell
* new classes for proteomics
* PyTorch


## Techniques and challenges for benchmarking methods

* realistic simulation studies
* cross-study validation
* benchmark datasets


## Discussion

### Emerging analytical methods and technologies

### Community needs for data structures, analysis methods, etc 


## References {.page_break_before}

<!-- Explicitly insert bibliography here -->
<div id="refs"></div>
