---
author-meta:
- "Kim-Anh L\xEA Cao"
- Aedin C Culhane
- Elana Fertig
- Jane Roe
bibliography:
- content/manual-references.json
date-meta: '2020-06-24'
header-includes: "<!--\nManubot generated metadata rendered from header-includes-template.html.\nSuggest improvements at https://github.com/manubot/manubot/blob/master/manubot/process/header-includes-template.html\n-->\n<meta name=\"dc.format\" content=\"text/html\" />\n<meta name=\"dc.title\" content=\"Mathematical Frameworks for Integrative Analysis of Multi-omics Biological Data\" />\n<meta name=\"citation_title\" content=\"Mathematical Frameworks for Integrative Analysis of Multi-omics Biological Data\" />\n<meta property=\"og:title\" content=\"Mathematical Frameworks for Integrative Analysis of Multi-omics Biological Data\" />\n<meta property=\"twitter:title\" content=\"Mathematical Frameworks for Integrative Analysis of Multi-omics Biological Data\" />\n<meta name=\"dc.date\" content=\"2020-06-24\" />\n<meta name=\"citation_publication_date\" content=\"2020-06-24\" />\n<meta name=\"dc.language\" content=\"en-US\" />\n<meta name=\"citation_language\" content=\"en-US\" />\n<meta name=\"dc.relation.ispartof\" content=\"Manubot\" />\n<meta name=\"dc.publisher\" content=\"Manubot\" />\n<meta name=\"citation_journal_title\" content=\"Manubot\" />\n<meta name=\"citation_technical_report_institution\" content=\"Manubot\" />\n<meta name=\"citation_author\" content=\"Kim-Anh L\xEA Cao\" />\n<meta name=\"citation_author_institution\" content=\"Melbourne Integrative Genomics, School of Mathematics and Statistics, University of Melbourne, Australia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0003-3923-1116\" />\n<meta name=\"twitter:creator\" content=\"@mixOmicsTeam\" />\n<meta name=\"citation_author\" content=\"Aedin C Culhane\" />\n<meta name=\"citation_author_institution\" content=\"Data Sciences, Dana-Farber Cancer Institute, Boston, MA, USA\" />\n<meta name=\"citation_author_institution\" content=\"Biostatsitics, Harvard TH Chan School of Public Health, Boston, MA, USA\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-1395-9734\" />\n<meta name=\"twitter:creator\" content=\"@AedinCulhane\" />\n<meta name=\"citation_author\" content=\"Elana Fertig\" />\n<meta name=\"citation_author_institution\" content=\"Department of Oncology, Sidney Kimmel Comprehensive Cancer Center, Johns Hopkins University School of Medicine, Baltimore, MD, USA\" />\n<meta name=\"citation_author_institution\" content=\"Department of Biomedical Engineering, Johns Hopkins University School of Medicine, Baltimore, MD, USA\" />\n<meta name=\"citation_author_institution\" content=\"Department of Applied Mathematics and Statistics, Johns Hopkins University Whiting School of Engineering, Baltimore, MD, USA\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0003-3204-342X\" />\n<meta name=\"twitter:creator\" content=\"@FertigLab\" />\n<meta name=\"citation_author\" content=\"Jane Roe\" />\n<meta name=\"citation_author_institution\" content=\"Department of Something, University of Whatever\" />\n<meta name=\"citation_author_institution\" content=\"Department of Whatever, University of Something\" />\n<meta name=\"citation_author_orcid\" content=\"XXXX-XXXX-XXXX-XXXX\" />\n<meta name=\"twitter:creator\" content=\"@XXX\" />\n<link rel=\"canonical\" href=\"https://BIRSBiointegration.github.io/whitePaper/\" />\n<meta property=\"og:url\" content=\"https://BIRSBiointegration.github.io/whitePaper/\" />\n<meta property=\"twitter:url\" content=\"https://BIRSBiointegration.github.io/whitePaper/\" />\n<meta name=\"citation_fulltext_html_url\" content=\"https://BIRSBiointegration.github.io/whitePaper/\" />\n<meta name=\"citation_pdf_url\" content=\"https://BIRSBiointegration.github.io/whitePaper/manuscript.pdf\" />\n<link rel=\"alternate\" type=\"application/pdf\" href=\"https://BIRSBiointegration.github.io/whitePaper/manuscript.pdf\" />\n<link rel=\"alternate\" type=\"text/html\" href=\"https://BIRSBiointegration.github.io/whitePaper/v/f9acb260522d80c981c837548fa7a5e1b1796a19/\" />\n<meta name=\"manubot_html_url_versioned\" content=\"https://BIRSBiointegration.github.io/whitePaper/v/f9acb260522d80c981c837548fa7a5e1b1796a19/\" />\n<meta name=\"manubot_pdf_url_versioned\" content=\"https://BIRSBiointegration.github.io/whitePaper/v/f9acb260522d80c981c837548fa7a5e1b1796a19/manuscript.pdf\" />\n<meta property=\"og:type\" content=\"article\" />\n<meta property=\"twitter:card\" content=\"summary_large_image\" />\n<link rel=\"icon\" type=\"image/png\" sizes=\"192x192\" href=\"https://manubot.org/favicon-192x192.png\" />\n<link rel=\"mask-icon\" href=\"https://manubot.org/safari-pinned-tab.svg\" color=\"#ad1457\" />\n<meta name=\"theme-color\" content=\"#ad1457\" />\n<!-- end Manubot generated metadata -->"
keywords:
- markdown
- publishing
- manubot
lang: en-US
manubot-clear-requests-cache: false
manubot-output-bibliography: output/references.json
manubot-output-citekeys: output/citations.tsv
manubot-requests-cache-path: ci/cache/requests-cache
title: Mathematical Frameworks for Integrative Analysis of Multi-omics Biological Data
...






<small><em>
This manuscript
([permalink](https://BIRSBiointegration.github.io/whitePaper/v/f9acb260522d80c981c837548fa7a5e1b1796a19/))
was automatically generated
from [BIRSBiointegration/whitePaper@f9acb26](https://github.com/BIRSBiointegration/whitePaper/tree/f9acb260522d80c981c837548fa7a5e1b1796a19)
on June 24, 2020.
</em></small>

## Authors



+ **Kim-Anh Lê Cao**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-3923-1116](https://orcid.org/0000-0003-3923-1116)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [mixOmicsTeam](https://github.com/mixOmicsTeam)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [mixOmicsTeam](https://twitter.com/mixOmicsTeam)<br>
  <small>
     Melbourne Integrative Genomics, School of Mathematics and Statistics, University of Melbourne, Australia
     · Funded by Grant National Health and Medical Research Council  Career Development fellowship (GNT1159458)
  </small>

+ **Aedin C Culhane**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-1395-9734](https://orcid.org/0000-0002-1395-9734)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [aedin](https://github.com/aedin)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [AedinCulhane](https://twitter.com/AedinCulhane)<br>
  <small>
     Data Sciences, Dana-Farber Cancer Institute, Boston, MA, USA; Biostatsitics, Harvard TH Chan School of Public Health, Boston, MA, USA
     · Funded by Chan Zuckerberg Initative, NIH, DoD (need to get grant IDs)
  </small>

+ **Elana Fertig**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-3204-342X](https://orcid.org/0000-0003-3204-342X)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [ejfertig](https://github.com/ejfertig)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [FertigLab](https://twitter.com/FertigLab)<br>
  <small>
     Department of Oncology, Sidney Kimmel Comprehensive Cancer Center, Johns Hopkins University School of Medicine, Baltimore, MD, USA; Department of Biomedical Engineering, Johns Hopkins University School of Medicine, Baltimore, MD, USA; Department of Applied Mathematics and Statistics, Johns Hopkins University Whiting School of Engineering, Baltimore, MD, USA
     · Funded by National Institute of Health, National Cancer Institute; National Institute of Health, National Institute of Dental and Craniofacial Research; Lustgarten Foundation; Emerson Foundation; Allegheny Health Network
  </small>

+ **Jane Roe**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [XXXX-XXXX-XXXX-XXXX](https://orcid.org/XXXX-XXXX-XXXX-XXXX)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [janeroe (PLEASE COPY/PASTE DO NOT EDIT THIS ONE)](https://github.com/janeroe (PLEASE COPY/PASTE DO NOT EDIT THIS ONE))
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [XXX](https://twitter.com/XXX)<br>
  <small>
     Department of Something, University of Whatever; Department of Whatever, University of Something
     · Funded by XX
  </small>



## Abstract {.page_break_before}




## Introduction to single cell and imaging multi-omics




## Current multi-omic technologies


## Challenges for interpretation

Need for technology-specific questions and analysis methods vs one size fits all data blender


## Case studies


### scNMT-seq as a case-study for epigenetic regulation

#### Overview and biological question

#### Computational challenges

* Identification of multi-omics signatures that characterise lineage, stage or both.
* Handling missing values
* Do epigenetic changes in some genomic contexts affect cell fate decision more than others? If so, how?

#### Methods for stats/maths analyses and results summary


### scRNA-seq + FISH as a case study for spatial transcriptomics

#### Overview and biological question

#### Computational challenges

* Can scRNA-seq data be overlaid onto seqFISH for resolution enhancement
* What is the minimal number of genes needed for data integration?
* Are there signatures of cellular co-localization or spatial coordinates in the non-spatial scRNA-seq data?

#### Methods for stats/maths analyses and results summary


### Spatial proteomics and cross-study analysis

#### Overview and biological question

#### Computational challenges

* Integrating partially-overlapping proteomic data collected on different patients with similar phenotypes
* Integration of spatial x-y coordinate co-location and co-expression
* Integration with other 'omics datasets (e.g., scRNA-seq) to support the results of these proteomic analyses
* Can we predict the spatial expression patterns of proteins measured on mass-tag but not measured in the MIBI-TOF data?
* What additional information can we learn about the different macrophage and immune populations in breast cancer by conducting integrated analyses of these datasets?

#### Methods for stats/maths analyses and results summary


## Overview of common analytical methods spanning technologies / case studies

* matrix factorization
* neural network / autoencoders


## Software strategies to enable analyses of multimodal single cell experiments

### Key questions 

* How should multimodal single cell data be managed for interactive and batch analyses?
* What methods will help software developers create scalable solutions for multimodal single cell analysis?
* How can we ensure that visualization methods that are central to multimodal single cell analysis
are usable by researchers with visual impairments?

### Data management strategies

* Abstract data type: "multiassay experiment".  This reflects the idea that each mode will
be characterized by a different collection of features on possibly non-overlapping collections
of samples.  The metadata on features should be clearly and conventionally defined.  For example,
genes and transcripts are enumerated using Ensembl catalog identifiers; regions of accessibility
are defined using genomic coordinates in a clearly specified reference build.  Metadata on
samples must include all relevant information on experimental conditions such as treatment,
protocol, and date of technical processing.
* Serializations and data access methods for
    * spatial transcriptomics
    * scNMT-seq ...


### Scalability strategies

### Reducing barriers to interpretable visualizations

[An overview of the issues with impaired color perception](https://www.vwvj.be/sites/default/files/zien/zien_-_uit_voorbije_vorming/kleurzinonderzoekextratips.pdf)

[US Government tools for accessibility](https://accessibility.18f.gov/tools/)

### Details of working components

|Type|Brief name|Description|URL|Author email|
|----|----------|-----------|---|------------|
|R data class|MultiAssayExperiment|unify multiple experiments|bioconductor.org|many|
|R package|Giotto|Spatial transcriptomics|...|...|
|python library|PyTorch|deep learning|...|...|



## Techniques and challenges for benchmarking methods

Outline:

* We must first define what we are benchmarking
    - recovery of cell types / clusters
	- discovery of relationships between data modalities, e.g. gene
      regulatory relationships observed between chromatin
      accessibility and gene expression
	- ...
* Strategies for benchmarking
    - simulation (and we can discuss the difficulties with simulating
      covariance structure across features and data modalities)
	- benchmarking datasets
    - cross-validation within study (and we can discuss issues in
      matching dimensions of latent space across folds). For this Mike
      has a lot of literature in Google Doc to include on papers that
      have performed either permutation or cross-validation to assess
      model performance.
    - cross-study validation (are relationships discovered in one
      dataset present in other datasets, potentially looking across
      single cell and bulk)



## Discussion

### Emerging analytical methods and technologies

### Community needs for data structures, analysis methods, etc 


## Glossary

| Consensus term | Synonyms | Description |
| -------------- | -------- | ----------- |
| Network | Graph | A set of *nodes*, representing objects of interest, linked by *edges*, representing specific relationships between nodes. |
| Node | Vertex | Element of interest in a network and linked to other nodes. For example: people, cells, proteins or genes. Nodes can have several properties called *attributes* like cell type or position. |
| Edge | Link | The relationship between 2 nodes in a network. For example: friendship in social networks, cells in contact in a spatial network, or gene-gene interactions in a gene regulatory network. |


## References {.page_break_before}

<!-- Explicitly insert bibliography here -->
<div id="refs"></div>
